src/journal/compress.c

```diff
@@ -231,6 +286,52 @@ int decompress_blob_lz4(const void *src, uint64_t src_size,
 #endif
 }
 
+
+int decompress_blob_zstd(const void *src, uint64_t src_size,
+                       void **dst, size_t *dst_alloc_size, size_t* dst_size, size_t dst_max) {
```

Comment:
strange indenting... either line it up with the `(`, or break it fully so that each argument gets its own line